pkg/cmd/grafana-cli/utils/grafana_path.go

```diff
@@ -2,35 +2,32 @@ package utils
 
 import (
 	"os"
+	"path/filepath"
 
 	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
 )
 
 func GetGrafanaPluginDir(currentOS string) string {
 	//currentOS := runtime.GOOS
 
-	if currentOS == "windows" {
-		return returnOsDefault(currentOS)
-	}
-
-	pwd, err := os.Getwd()
-
-	if err != nil {
-		logger.Error("Could not get current path. using default")
-		return returnOsDefault(currentOS)
-	}
-
-	if isDevenvironment(pwd) {
+	if isDevenvironment() {
 		return "../data/plugins"
 	}
 
 	return returnOsDefault(currentOS)
 }
 
-func isDevenvironment(pwd string) bool {
+func isDevenvironment() bool {
 	// if ../conf/defaults.ini exists, grafana is not installed as package
 	// that its in development environment.
-	_, err := os.Stat("../conf/defaults.ini")
+	ex, err := os.Executable()
+	if err != nil {
+		logger.Error("Could not get executable path. Assuming non dev environment.")
+		return false
+	}
+	exPath := filepath.Dir(ex)
+	defaultsPath := filepath.Join(exPath, "../conf/defaults.ini")
```

Comment:
```suggestion
	defaultsPath := filepath.Join(exPath, "../../conf/defaults.ini")
```

This hasn't worked as expected for a long time since the paths assume the cli is placed in `bin` while we build it into `bin/linux-amd64`

Using `os.Executable` is a great idea :)

Reply:
Hmm what about `bin/grafana-cli` then? We are building the current architecture directly in `bin/` right? And then also inside a directory for each architecture? Maybe that is why it was PWD before as the binary is build in different places?

Reply:
https://github.com/grafana/grafana/blob/master/build.go#L466

maybe. we should have a subfolder for dev to make it more consistent. `bin/dev/grafana-cli`
I'm no sure whats the best solution here :)

Reply:
Another use case that might not be obvious here is people using the tarball installation on linux. In those cases, the binaries will be placed in `bin` (so only one folder level then)

Might make sense to check `../conf/defaults.ini` first then `../../conf/defaults.ini`